app/helpers/mailbox_helper.rb

```diff
@@ -2,6 +2,6 @@ module MailboxHelper
   def unread_messages_count
     # how to get the number of unread messages for the current user
     # using mailboxer
-    mailbox.inbox(unread: true).count(:id, distinct: true)
+    mailbox.inbox(unread: true).count(:id)
```

Comment:
When we clicked the inbox button we got an error which was referring to this line. Only way the website loaded was if the distinct: true was removed.